src/Sparrow/LowMemory/LowMemoryNotification.cs

```diff
@@ -323,12 +318,7 @@ public bool IsLowMemory(MemoryInfoResult memInfo, SmapsReader smapsReader)
                 isLowMemory = IsAvailableMemoryBelowThreshold(memInfo);
             }
 
-            if (PlatformDetails.RunningOnPosix == false)
-            {
-                // this is relevant only on Windows
-                var commitChargePlusMinSizeToKeepFree = memInfo.CurrentCommitCharge + GetCommitChargeThreshold(memInfo);
-                isLowMemory |= memInfo.TotalCommittableMemory <= commitChargePlusMinSizeToKeepFree;
-            }
+            isLowMemory |= MemoryInformation.IsEarlyOutOfMemory(memInfo);
```

Comment:
Not sure if this is correct, because we will have a low memory exactly when first EOOM can be thrown and we probably want to do it a bit earlier to give a chance to cleanup a bit and to not throw the EOOM?

Reply:
I suggest adding another configuration: LowMemoryCommitLimitInMb (by default set to 3 times the MaxFreeCommittedMemoryToKeepInMb).
For a 16GB machine:
MaxFreeCommittedMemoryToKeepInMb: 128MB, lower than this will throw EOOM
LowMemoryCommitLimitInMb: 384MB, lower than this, will trigger low memory notification

Reply:
That sounds fine, but it should be at least 512MB by default or 5% (smallest).